src/main/java/org/opensearch/knn/quantization/models/quantizationOutput/BinaryQuantizationOutput.java

```diff
@@ -63,4 +63,16 @@ public boolean isPrepared(int vectorLength) {
     public byte[] getQuantizedVector() {
         return quantizedVector;
     }
+
+    /**
+     * Returns a copy of the quantized vector.
+     *
+     * @return a copy of the quantized vector byte array.
+     */
+    @Override
+    public byte[] getQuantizedVectorCopy() {
```

Comment:
[Not a blocker]
Lets add the documentation of the `getQuantizedVector` that it gives the same reference object in return. This will ensure that person using the function knows how to use these functions.